handleData.js

```diff
@@ -119,11 +127,52 @@ var Character = function(realName, gameName, level, gameClass, race) {
     this.characterLevel = level;
     this.characterClass = gameClass;
     this.characterRace = race;
+    this.inventory = [];
+    this.money = { "Copper Pieces"      : 0,
+                   "Silver Pieces"      : 0,
+                   "Gold Pieces"        : 0,
+                   "Platinum Pieces"    : 0,
+                   "Electrum Pieces"    : 0};
```

Comment:
Some other thoughts on the trailing comma argument: https://dnlserrano.github.io/2015/07/29/a-case-for-trailing-commas/

It's *mostly* a personal preference btw, but I personally prefer the consistency of it.